test_hello_worlds/hello.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
```

Comment:
This file contains at least one console log. Please remove any present. - additions.length